.github/workflows/ci-pull-request.yaml

```diff
@@ -29,3 +29,12 @@ jobs:
         uses: actions/checkout@v3
       - name: Lint
         run: make lint
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Test
+        run: make test
```

Comment:
this currently relies on the default installed terraform version (currently 1.5.0) Argumably we should be testing against more recent versions of terraform and perform a matrix build